src/Simplex/Chat.hs

```diff
@@ -2799,7 +2812,7 @@ processAgentMessageConn user@User {userId} corrId agentConnId agentMessage = do
             _ -> pure ()
         SWITCH qd phase cStats -> do
           toView $ CRContactSwitch user ct (SwitchProgress qd phase cStats)
-          when (phase /= SPConfirmed) $ case qd of
+          when (phase `notElem` [SPConfirmed, SPSecured]) $ case qd of
```

Comment:
at this point I'd rather we write what it should be for event to be created

Reply:
if we were to add to this release coalescing multiple control items into one - it's been long due and very annoying for groups - maybe we should make all events here